src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -66,45 +66,47 @@ class TutorialDirectoryUi extends React.Component {
   constructor(props) {
     super(props);
 
+    // TODO: Enable the tabs once we have instructions
+    // and sample code snippets to instruct users to add data
     this.tabs = [
-      {
-        id: ALL_TAB_ID,
-        name: this.props.intl.formatMessage({
-          id: 'home.tutorial.tabs.allTitle',
-          defaultMessage: 'All',
-        }),
-      },
-      {
-        id: 'logging',
-        name: this.props.intl.formatMessage({
-          id: 'home.tutorial.tabs.loggingTitle',
-          defaultMessage: 'Logs',
-        }),
-      },
-      {
-        id: 'metrics',
-        name: this.props.intl.formatMessage({
-          id: 'home.tutorial.tabs.metricsTitle',
-          defaultMessage: 'Metrics',
-        }),
-      },
-      {
-        id: 'security',
-        name: this.props.intl.formatMessage({
-          id: 'home.tutorial.tabs.securitySolutionTitle',
-          defaultMessage: 'Security',
-        }),
-      },
-      {
-        id: SAMPLE_DATA_TAB_ID,
-        name: this.props.intl.formatMessage({
-          id: 'home.tutorial.tabs.sampleDataTitle',
-          defaultMessage: 'Sample data',
-        }),
-      },
+      //{
+      //  id: ALL_TAB_ID,
+      //  name: this.props.intl.formatMessage({
+      //    id: 'home.tutorial.tabs.allTitle',
+      //    defaultMessage: 'All',
+      //  }),
+      //},
+      //{
+      //  id: 'logging',
+      //  name: this.props.intl.formatMessage({
+      //    id: 'home.tutorial.tabs.loggingTitle',
+      //    defaultMessage: 'Logs',
+      //  }),
+      //},
+      //{
+      //  id: 'metrics',
+      //  name: this.props.intl.formatMessage({
+      //    id: 'home.tutorial.tabs.metricsTitle',
+      //    defaultMessage: 'Metrics',
+      //  }),
+      //},
+      //{
+      //  id: 'security',
+      //  name: this.props.intl.formatMessage({
+      //    id: 'home.tutorial.tabs.securitySolutionTitle',
+      //    defaultMessage: 'Security',
+      //  }),
+      //},
+      //{
+      //  id: SAMPLE_DATA_TAB_ID,
+      //  name: this.props.intl.formatMessage({
+      //    id: 'home.tutorial.tabs.sampleDataTitle',
+      //    defaultMessage: 'Sample data',
+      // }),
+      //},
```

Comment:
If we're going to revert this commit and make adjustments in the future, why not completely remove this code instead of commenting it out?

Reply:
Hmmm will do